python/lsst/drp/tasks/assemble_cell_coadd.py

```diff
@@ -66,6 +66,15 @@ class AssembleCellCoaddConnections(
         multiple=True,
     )
 
+    artifactMasks = Input(
+        doc="Artifact masks to be applied to the input warps",
+        name="artifactMask",
```

Comment:
I've kept the name as just `artifactMask` after our discussions.

Reply:
`artifact_mask` is fine.  I think I used `warp_mask` as an indication of the coordinate system the mask was in, so maybe `warp_artifact_mask` or even `compare_warp_mask` may be worth considering.

I'm glad John and others have paid more attention to the DMTN-172 conventions than I have (at least in code reviews :slightly_smiling_face:); I lost steam on actually getting all of that RFC-approved but it's better to adhere to informal naming conventions than none.

Reply:
`compare_warp_artifact_mask`. No one said naming is easy.